.github/workflows/integration-tests.yml

```diff
@@ -155,10 +156,13 @@ jobs:
         #type: [Release, Debug]
         type: [Release]
         platform:
-          [
-            { name: ios, build-configuration: simulator },
-            { name: catalyst, build-configuration: catalyst },
-          ]
+          - name: ios
+            build-configuration: simulator
+          - name: catalyst
+            build-configuration: catalyst
+    env:
+      # Pin the Xcode version
+      DEVELOPER_DIR: /Applications/Xcode_12.5.1.app
```

Comment:
The main change is this line pinning the Xcode version. We need to unpin this version once we figure out the root cause, ensuring we'll be compatible with end-user environments.